pkg/parser/docker/parser.go

```diff
@@ -38,7 +38,7 @@ func (p *Parser) Parse(_ string, fileContent []byte) ([]model.Document, error) {
 
 	parsed, err := parser.Parse(reader)
 	if err != nil {
-		return nil, errors.Wrap(err, "Failed to parse Dockerfile")
+		return nil, errors.Wrap(err, "failed to parse Dockerfile")
```

Comment:
why in some cases it starts with lowercase and in other it was changed to start with uppercase?

Reply:
@ruigomescx  There's a linter rule that errors strings should not start with uppercase letter nor end with punctuation or new line. So whenever there's `errors.New` or `errors.Wrap` I'll put them lowercase because they'll be logged uphill by another `log.Err(err).Msg` call.